### PR TITLE
PRA-170 Drag-and-drop vertical

### PR DESCRIPTION
**Pull request by AnnaSolovykh**
## Requirements

Drag and drop functionality between the recipe options.

## Notes

Footer is commented out in this PR because of its central position. After merging previous PRs with the corrected position I will be able to open it.

Do npm install for testing 

## Test Steps

Try to drag and drop the recipes. 

## UI/UX Outcome

Screen Shot 2023-08-24 at 17 24 52

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

### CODE REVIEW

**Review by AnnaPestova1 — APPROVED** [review]
works as expected

**Review by AnnaPestova1 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -53,6 +54,7 @@
   },
   "devDependencies": {
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
+    "@types/react-beautiful-dnd": "^13.1.4"
```
Comment (rebekahcallkacz) [inline package.json:57]: BLOCKING: You need a comma at the end of this line

**Review by rebekahcallkacz — APPROVED** [review]
Don't merge this until you add that comma (see other comment), but I'm going to go ahead and approve since I won't need to re-review this after you do that.

**Comment by AnnaSolovykh** [comment]
Oops 😬 Merging conflict error... thank you for noticing!

File: src/components/Planner/Planner.tsx [hunk 2]
```diff
@@ -1,7 +1,98 @@
...
+              </Box>
+            )}
+          </Droppable>
+        </GridItem>
+        <GridItem colSpan={{ base: 1, md: 6 }}>
+          <Grid
+            h={{ base: "auto", md: "200px" }}
+            templateRows={{ base: "repeat(4, 1fr)", md: "repeat(4, 1fr)" }}
+            templateColumns={{ base: "1fr", md: "repeat(7, 1fr)" }}>
+            {["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"].map(
```
[the remark this answers is not in the document]
Comment (AnnaSolovykh) [reply src/components/Planner/Planner.tsx:77]: Done!

**Pull request merged**